src/main/scala/org/bytedeco/sbt/javacpp/JavaCppPlugin.scala

```diff
@@ -1,34 +1,40 @@
 package org.bytedeco.sbt.javacpp
 
-import scala.language.postfixOps
-import sbt._
+import org.bytedeco.javacpp.tools.Builder
```

Comment:
Stupid question: this `Builder` , which is a plugin-level JavaCPP `Builder` with a set version based on 

https://github.com/bytedeco/sbt-javacpp/blob/cc93495947419c461df5473ac97a447e0a83405e/build.sbt#L26

, isn't necessarily going to support the classes brought in by `""org.bytedeco" % "javacpp" % Versions.javaCppVersion jar"`, which is a project-level, right?

Reply:
Because `Versions.javaCppVersion` is deducted from the filename from which the `Builder` class is loaded, it should be the same as what is set in `set-javacpp/build.sbt`. So the same version of JavaCpp library should be used by the user project and the plugin, unless the user specified a different version in `build.sbt` instead of `project/plugins.sbt`. I updated the README documentation on how to specify a different JavaCPP version. It is a breaking change though. A consistency check can be added before JavaCPP build, and print a warning if the library versions do not match.

Reply:
@thirstycrow Does that mean we can remove that line with JavaCPP 1.4.3 from `sbt-javacpp/built.sbt`?

Reply:
No, the JavaCPP library is required by both the user project (to define the JavaCPP classes) and the sbt-javacpp plugin (to build the JNI codes).

Reply:
Ok, so let's set them to the same version?